easybuild/easyconfigs/c/Clang/Clang-8.0.0-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,56 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'Clang'
+version = '8.0.0'
+
+homepage = 'http://clang.llvm.org/'
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
```

Comment:
@boegel Was wondering why this is done like this (rather than listing GCCcore as a dependency like we do for other compilers). If we want to upgrade Clang to a toolchain this would have to change.

Reply:
@ocaisa Just using `GCCcore` as a dep is very different than using it as a toolchain w.r.t. build environment, and since we're building `Clang` from source that's important I think...

To use this as a toolchain, we can define a `Clang-8.0.0-GCC-8.2.0-2.32.eb` bundle, which also drags in `binutils`?

Reply:
I'll see if I can get Flang built (not easy, they have forks of everything) and then maybe look into an LLVM toolchain.

Reply:
What about name conflicts? Clang and LLVM are both "taken" already.

Reply:
Let's see if we can get a working build first and worry about that later. I'm concerned that the Flang forks of LLVM etc. could override the Clang installation if we are not careful.